lstchain/pointing/pointings.py

```diff
@@ -0,0 +1,86 @@
+import os
+import numpy as np
+from ctapipe.core import Component
+from ctapipe_io_lst.containers import LSTDriveContainer
+from ctapipe.core.traits import Unicode, Int
+
+__all__ = [
+    'PointingPosition'
+]
+
+
+class PointingPosition(Component):
+    """
+      Pointion position of telescopes
+
+    """
+    drive_path = Unicode(
+        '',
+        allow_none=True,
+        help='Path to the LST drive report file'
+    ).tag(config=True)
+
+    tel_id = Int(
+        0,
+        help='id of the telescope to take drive report for'
+    ).tag(config=True)
+
+    def _read_drive_report(self):
+        """
+        Reading drive reports
+
+        Returns:
+        data:`~astropy.table.Table`
+             A table of drive reports
+
+        """
+        from astropy.io import ascii
```

Comment:
pep8 recommends making all imports at the beginning of the file

Reply:
Ok. We'll put it at the beginning